src/legend_data_monitor/plotting.py

```diff
@@ -150,13 +157,15 @@ def make_subsystem_plots(subsystem: subsystem.Subsystem, plots: dict, plt_path:
         plot_structure = PLOT_STRUCTURE[plot_settings["plot_structure"]]
         utils.logger.debug("Plot structure: " + plot_settings["plot_structure"])
 
+        # plotting
         par_dict_content = plot_structure(data_analysis, plot_info, pdf)
 
+        # saving dataframe for each parameter
```

Comment:
I wanted to move it even before calling `plot_structure()`, in the end we are not calling `plot_status()` from dashboard (or at least not in the near future?) and it does not seem to affect parameters plotting.

Reply:
ok but better to leave it after the `reset_index()` line as it was, there is no consequence of the rest of the code ofc since you're just saving the dataframe. But better to save it with right columns/index